### DISCUSSION TITLE
geemap and leafmap installation

### DISCUSSION THREAD

**Discussion by Rockaholics** [giswqs/geog-312#33] [Q&A] [answered]
Hello sir, I am following your tutorial on YouTube and I'm getting the following error message during installation of Geemap and Leafmap;
An unexpected error has occurred. Conda has prepared the above report.
If you suspect this error is being caused by a malfunctioning plugin,
consider using the --no-plugins option to turn off plugins.

Example: conda --no-plugins install <package>

Alternatively, you can set the CONDA_NO_PLUGINS environment variable on
the command line to run the command without plugins enabled.

Example: CONDA_NO_PLUGINS=true conda install <package>

**Comment by giswqs** [comment, marked as answer]
If conda does not work for you, you can try pip. 
```bash
pip install geemap leafmap
```

**Reply by Rockaholics** [reply]
Thank you.
